lib/pdf_fill/forms/common_ptsd.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module PdfFill
+  module Forms
+    module CommonPtsd
+      include FormHelper
+
+      def expand_ssn(hash)
+        ssn = hash['veteranSocialSecurityNumber']
+        return if ssn.blank?
```

Comment:
Should this return `hash` unchanged if `ssn` is blank? As it's currently written this will set `@form_data` to `nil` if `ssn` is blank.

Reply:
yes, it should.  thanks.